backends/bmv2/base_test.py

```diff
@@ -246,22 +248,19 @@ def __exit__(self, exc_type, exc_value, tb):
         if not issubclass(exc_type, P4RuntimeWriteException):
             # let unexpected exceptions pass through
             return False
-        self.exception = exc_value  # store for later retrieval
+        self.exception = exc_value # store for later retrieval
```

Comment:
Having formatting that complies with Mihai's request and the two space rule is unfortunately quite difficult. I have not been able to find a configuration that does both (Aligning comments as well as maintaining a minimum distance). 

 Personally, I prefer to have no inline comments at all since they often run over the maximum character length.

Reply:
https://github.com/google/yapf/issues/675

This is the issue I am facing. But other formatters are also not better at this. It might be less headache to forbid inline comments in general.